jdi-light-mobile-tests/src/main/java/pseudo/site/webpage/ExtendedWebPage.java

```diff
@@ -0,0 +1,450 @@
+    @XPath("//*[@class='webElementPublicXPath']")
+    public WebElement webElementPublicXPath;
+
+    @FindBy(id = "listWebElementPackage")
+    List<WebElement> listWebElementPackage;
+    @FindBy(id = "listWebElementPublic")
+    public List<WebElement> listWebElementPublic;
+    @FindBy(id = "listWebElementPrivate")
+    private List<WebElement> listWebElementPrivate;
+    @UI(".listWebElementPackageUI")
+    List<WebElement> listWebElementPackageUI;
+    @UI(".listWebElementPublicUI")
+    public List<WebElement> listWebElementPublicUI;
+    @Css(".listWebElementPackageCss")
+    List<WebElement> listWebElementPackageCss;
+    @XPath("//*[@class='listWebElementPublicXPath']")
+    public List<WebElement> listWebElementPublicXPath;
+
+    @FindBy(id = "uielementPackage")
+    UIElement uielementPackage;
+    @FindBy(id = "uielementPublic")
+    public UIElement uielementPublic;
+    @FindBy(id = "uielementPrivate")
+    private UIElement uielementPrivate;
+    @UI(".uielementPackageUI")
+    UIElement uielementPackageUI;
+    @UI(".uielementPublicUI")
+    public UIElement uielementPublicUI;
+    @Css(".uielementPackageCss")
+    UIElement uielementPackageCss;
+    @XPath("//*[@class='uielementPublicXPath']")
+    public UIElement uielementPublicXPath;
+
+    @FindBy(id = "buttonPackage")
+    Button buttonPackage;
+    @FindBy(id = "buttonPublic")
+    public Button buttonPublic;
+    @FindBy(id = "buttonPrivate")
+    private Button buttonPrivate;
+    @UI(".buttonPackageUI")
+    Button buttonPackageUI;
+    @UI(".buttonPublicUI")
+    public Button buttonPublicUI;
+    @Css(".buttonPackageCss")
+    UIElement buttonPackageCss;
+    @XPath("//*[@class='buttonPublicXPath']")
+    public UIElement buttonPublicXPath;
+
+    @FindBy(id = "listUIElementPackage")
+    List<UIElement> listUIElementPackage;
+    @FindBy(id = "listUIElementPublic")
+    public List<UIElement> listUIElementPublic;
+    @FindBy(id = "listUIElementPrivate")
+    private List<UIElement> listUIElementPrivate;
+    @UI(".listUIElementPackageUI")
+    List<UIElement> listUIElementPackageUI;
+    @UI(".listUIElementPublicUI")
+    public List<UIElement> listUIElementPublicUI;
+    @Css(".listUIElementPackageCss")
+    List<UIElement> listUIElementPackageCss;
+    @XPath("//*[@class='listUIElementPublicXPath']")
+    public List<UIElement> listUIElementPublicXPath;
+
+    @FindBy(id = "listButtonPackage")
+    List<Button> listButtonPackage;
+    @FindBy(id = "listButtonPublic")
+    public List<Button> listButtonPublic;
+    @FindBy(id = "listButtonPrivate")
+    private List<Button> listButtonPrivate;
+    @UI(".listButtonPackageUI")
+    List<Button> listButtonPackageUI;
+    @UI(".listButtonPublicUI")
+    public List<Button> listButtonPublicUI;
+    @Css(".listButtonPackageCss")
+    List<Button> listButtonPackageCss;
+    @XPath("//*[@class='listButtonPublicXPath']")
+    public List<Button> listButtonPublicXPath;
+
+    @FindBy(id = "webListPackage")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5217753)